src/GitHub.App/SampleData/PullRequestCreationViewModelDesigner.cs

```diff
@@ -9,27 +14,45 @@ public class PullRequestCreationViewModelDesigner : BaseViewModelDesigner, IPull
     {
         public PullRequestCreationViewModelDesigner()
         {
-            Branches = new List<string>()
+            //TODO:Make String into an IBranch
+            Branches = new ReactiveList<IBranch>
             {
-                "don/stub-ui",
-                "feature/pr/views",
-                "release-1.0.17.0"
+               new Branch {Name = "don/stub-ui"},
+               new Branch {Name = "feature/pr/views"},
+               new Branch {Name = "release-1.0.17.0"}
```

Comment:
Make sure this follows the code design guidelines - spaces before `{` and `}` (same for code below)